Book-A-Desk.Api.Tests/EmailNotificationTests.fs

```diff
@@ -0,0 +1,63 @@
+module Book_A_Desk.Api.Tests.EmailNotificationTests
+
+open System
+open Book_A_Desk.Api
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.Office.Domain
+open Xunit
+
+[<Fact>]
+let ``GIVEN A booking WHEN calling to SendEmailNotification THEN Email must be sent`` () = async {
```

Comment:
This test isn't valid because you don't really test that the email is sent

Reply:
To Do: Remove current test and check different functions in the domain.